src/addons/monkey-patching.md

```diff
@@ -1,24 +1,30 @@
-# Monkey patching
+# Monkey Patching
 
-___
+<p class="author center">Courtesy of RavenAscendant#7504</p>
 
-Courtesy of RavenAscendant#7504
+---
 
-All code snippets used in the guide are licensed under a Creative Commons Attribution-NonCommercial-ShareAlike 3.0 Unported License
+<p class="center">
+All code snippets used in the guide are licensed under a Creative Commons <br>
+Attribution-NonCommercial-ShareAlike 3.0 Unported License
+</p>
 
-This guide is specifically written for modding [Anomaly](https://www.moddb.com/mods/stalker-anomaly). Much of it will apply with minor changes to any STALKER modding. The concept is not limited to STALKER or even lua, Minecraft modding makes extensive use of monkey patching in Java.
+## About
 
-Monkey patching is the practice of modifying code at run time. Instead of modifying the script file directly the modifications are done in memory by a second script. The primary reason for doing this in Anomaly is compatibility. If two addons are distributed with an edited version of the same Anomaly game script the two addons will be incompatible without  patch. Worse than incompatible they will most likely cause crashes if installed together. Monkey patching doesn’t guarantee compatibility, if two addons change the same thing in different ways it is unlikely to behave properly, however the likelihood of crashing is significantly reduced.
+Monkey Patching is the practice of modifying code at run time. Instead of modifying the script file directly the modifications are done in memory by a second script. The primary reason for doing this in Anomaly is compatibility. If two addons are distributed with an edited version of the same Anomaly game script the two addons will be incompatible without  patch. Worse than incompatible they will most likely cause crashes if installed together. Monkey Patching doesn’t guarantee compatibility, if two addons change the same thing in different ways it is unlikely to behave properly, however the likelihood of crashing is significantly reduced.
```

Comment:
```suggestion
Monkey patching is the practice of modifying code at run time. Instead of modifying the script file directly the modifications are done in memory by a second script. The primary reason for doing this in Anomaly is compatibility. If two addons are distributed with an edited version of the same Anomaly game script the two addons will be incompatible without  patch. Worse than incompatible they will most likely cause crashes if installed together. Monkey patching doesn’t guarantee compatibility, if two addons change the same thing in different ways it is unlikely to behave properly, however the likelihood of crashing is significantly reduced.
```

Reply:
Camel Case in article name is fine, but there is nothing special about the name to make it capitalized in text